client/lib/plans/features-list.tsx

```diff
@@ -319,7 +322,10 @@ export type FeatureObject = {
 	getCompareSubtitle?: () => TranslateResult;
 	getIcon?: () => string | { icon: string; component: MemoExoticComponent< any > } | JSX.Element;
 	isPlan?: boolean;
+	isAddOn?: boolean;
```

Comment:
If we are going with this pattern, then let's add a comment here explaining how this is the used and what use case is serving. Let's also clarify that it is not mean to be a "classification" property (as there are a mixture of add-on patterns - we are just introducing another).

Reply:
[3489e21](https://github.com/Automattic/wp-calypso/pull/79041/commits/3489e21b833e80b8e0ea6968c9c028fb038d570d)